app/controllers/RiderStageConnectionController.java

```diff
@@ -2,8 +2,10 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import models.Race;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'models.Race'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)